tests/Feature/AboutTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Http\Controllers\AboutController;
+use App\Models\Contributor;
+use App\Support\FetchContributors;
+use Mockery\MockInterface;
+use Tests\TestCase;
+
+class AboutTest extends TestCase
+{
+    public function test_it_has_contributors(): void
+    {
+        $contributors = [
+            new Contributor(...[
+                'id' => 6905297,
+                'name' => 'Brent',
+                'url' => 'https://github.com/brendt',
+                'contributions' => ['Frontend', 'Backend'],
+            ]),
+        ];
+
+        $this->mock(FetchContributors::class, function (MockInterface $mock) use ($contributors) {
+            $mock->shouldReceive('getJson')->andReturn([]);
```

Comment:
This line is not really needed, we can mock just the outer call.
```suggestion
```

Reply:
It's just there to avoid loading the contents of the json file into memory.

Reply:
But it's not being loaded. You're mocking the whole class and actual methods are not being called in this test.